plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -415,6 +418,47 @@ def update_client(self, id, clientrep, realm="master"):
             self.module.fail_json(msg='Could not update client %s in realm %s: %s'
                                       % (id, realm, str(e)))
 
+    def get_authz_authorization_scope_by_name(self, name, client_id, realm):
```

Comment:
It seems I accidentally put this in between *_client methods. I'll fix that along the other fixes this PR may require.